_docs/how_to_run_acceptance_tests.md

```diff
@@ -0,0 +1,172 @@
+  It is a good idea to export the `BEAKER_HYPERVISOR` variable in your shell configuration.
+* `BEAKER_DESTROY` Should the test environment be removed at the end of a test
+  * `BEAKER_DESTROY=onpass` Only removes the environment if everything passes, allowing review of the system in the state it was in when the test suite failed.
+  * `BEAKER_DESTROY=no` Always leave an artifact, which likely will break future runs without cleanup.
+  * `BEAKER_DESTROY=yes` _(DEFAULT)_ always clean up.
+* `BEAKER_PROVISION` Should we ensure a clean system is built to run the suite.
+  * If `BEAKER_DESTROY` is set to no, `BEAKER_PROVISION=yes` will fail the run (because of the existing box/container)
+  * If `BEAKER_DESTROY` is set to no, `BEAKER_PROVISION=no` will run the test suite against the system anyways, which may cause environmental issues if the test suite doesn't perfectly put things to a default state.
+
+* `BEAKER_SETFILE` What should we be testing.  Beaker will call beaker-hostgenerator to create a defaultconfiguration based on this, using the known configurations for Vox Pupuli.
+* `BEAKER_PUPPET_COLLECTION` What implementation and version of puppet are we testing against
+
+
+### Listing Platforms
+
+Installed as part of the voxpupuli test suite is `metadata2gha`, which as its name suggests, is a helper to generate a json blob containing the variables for all supported oses.
+
+```shell
+# From the base directory of the module
+$ bundle exec metadata2gha | tail -n 1 | cut -d= -f2- | jq
+[
+# ... truncated for length
+  {
+    "name": "Puppet 8 - Debian 12",
+    "env": {
+      "BEAKER_PUPPET_COLLECTION": "puppet8",
+      "BEAKER_SETFILE": "debian12-64{hostname=debian12-64-puppet8}"
+    }
+  },
+  {
+    "name": "Puppet 7 - Debian 12",
+    "env": {
+      "BEAKER_PUPPET_COLLECTION": "puppet7",
+      "BEAKER_SETFILE": "debian12-64{hostname=debian12-64-puppet7}"
+    }
+  },
+# ... truncated for length
+  {
+    "name": "OpenVox 8 - Rocky 9",
+    "env": {
+      "BEAKER_PUPPET_COLLECTION": "openvox8",
+      "BEAKER_SETFILE": "rocky9-64{hostname=rocky9-64-openvox8}"
+    }
+  },
+  {
+    "name": "OpenVox 7 - Rocky 9",
+    "env": {
+      "BEAKER_PUPPET_COLLECTION": "openvox7",
+      "BEAKER_SETFILE": "rocky9-64{hostname=rocky9-64-openvox7}"
+    }
+  },
+# ... truncated for length
+]
+```
+
+This returns names that show puppet implementation (openvox or puppet) / language version, and operating system / version.
+It also provides the environment variables needed to be applied to run beaker for this combination.
+
+### Running Beaker
+
+At this point, everything is ready to run beaker.
+
+Because everything is an environment variable, there are some options on running things.
+
+1. export variables, then run `bundle exec rake beaker`
+   ```shell
+   export BEAKER_PUPPET_COLLECTION="openvox8"
+   export BEAKER_SETFILE="ubuntu2404-64{hostname=ubuntu2404-64-openvox8}"
```

Comment:
If you don't have a domain then I'm not sure what the point of the `hostname` is. This should have `ubuntu2404-64` as the hostname and I think that's fine.
```suggestion
   export BEAKER_SETFILE="ubuntu2404-64"
```

Reply:
I was trying to show it as metadata2gha was using it, but given that both you and @bastelfreak are poking the hostname/collection, I'll have to think about it some.